docs/messaging/gcp-pubsub.md

```diff
@@ -16,6 +16,7 @@ For Google Cloud Pub/Sub, the following additional attributes are defined:
 <!-- semconv messaging.gcp_pubsub(full,tag=tech-specific-gcp-pubsub) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| [`messaging.consumer.group.name`](../attributes-registry/messaging.md) | string | The name of the Pub/Sub topic subscription. | `my-group`; `indexer` | Conditionally Required: On consumer spans. |
```

Comment:
@alevenberg  @hongalex @samwright how do you feel about this?

totally open to other naming suggestions - it sounds like we have two  popular terms - "consumer group" and "subscription" to describe load/balancing + fan-out. Some systems even use both notions ([RocketMQ](https://rocketmq.apache.org/docs/domainModel/07consumergroup)).